iac/example.yml

```diff
@@ -0,0 +1,33 @@
+# A sample workflow which checks out your Infrastructure-as-Code Configuration files,
```

Comment:
Infrastructure-as-Code shouldn't be hyphenated I don't think. I don't think we do so elsewhere and common usage isn't to do so.